project-template.md

```diff
@@ -1,8 +1,14 @@
 # <<PROJECT NAME>>
 
-## Description
+## Background and description
 
-Describe the goals, objectives, and requirements for the project. This include the motivations for starting the project now, as opposed to later.
+Add any background that may be needed to introduce the scope of this project.
+
+### Current challenges
+List the current challenges that this project aims to solve. Focus on the problem here, and how it affects users, rather than the solution.
```

Comment:
```suggestion
List the current challenges that this project aims to solve. Focus on the problem here, how it affects users and what the downsides are if nothing is done, rather than the solution.
```